javascript/javascript1/week1/js1homework2.js

```diff
@@ -0,0 +1,34 @@
+//Excercise 3
+
+function HousePrice(houseDimension, gardenSize, actualPricePaid, userName) {
```

Comment:
Impressive 👏 I thought about using arrays where the first value would be related to Peter, and the second related to Julia. But this is great also. 🌸